clients/client-go/integrationtest/integration_test.go

```diff
@@ -1,192 +0,0 @@
-package integrationtest
-
-import (
-	"encoding/json"
-	"testing"
-	"time"
-
-	"github.com/taskcluster/slugid-go/slugid"
-	"github.com/taskcluster/taskcluster-base-go/jsontest"
-	tcclient "github.com/taskcluster/taskcluster/v25/clients/client-go"
-	"github.com/taskcluster/taskcluster/v25/clients/client-go/tcindex"
-	"github.com/taskcluster/taskcluster/v25/clients/client-go/tcqueue"
-)
-
-// This is a silly test that looks for the garbage index namespace, to somewhat
-// validate that the index client was generated correctly.
-//
-// Note, no credentials are needed.
-func TestGarbageNamespaces(t *testing.T) {
-	Index := tcindex.NewFromEnv()
-	n, err := Index.ListNamespaces("garbage", "", "")
-	if err != nil {
-		t.Fatalf("%v\n", err)
-	}
-	t.Logf("Namespaces: %#v", n.Namespaces)
-}
```

Comment:
I think this was a useful test. Did you mean to delete it? It ensures that after we've built the client with the code generation, that it can perform at least a simple API request, to list index namespaces.